.gitattributes

```diff
@@ -1,8 +1,12 @@
 * text eol=lf
-*.bat text eol=crlf
```

Comment:
Hm ... why changing that? `*.bat` are 100% Windows specific, where `crlf` rules

Reply:
Some how linux is unhappy with `modified:   gradlew.bat`

Reply:
We probably need to find out why, Windows users might not like it since not all editors on Windows support Linux style, mind if I take a look tomorrow? Thank you.

Reply:
Absolutely, I'll hold this PR and I'll take a look as well. Thanks @reta !

Reply:
@saratvemulapalli totally forgot about this one, sorry, looking ...

Reply:
Ok, nonetheless it looks a bit weird, the binary does not hurt, let's try it out

Reply:
Yeah lets give it a shot, feels like all platforms are happy. We can always change it.